src/vs/workbench/services/untitled/common/untitledTextEditorService.ts

```diff
@@ -123,6 +128,9 @@ export class UntitledTextEditorService extends Disposable implements IUntitledTe
 
 	declare readonly _serviceBrand: undefined;
 
+	private readonly _onDidChangeContent = this._register(new Emitter<IUntitledTextEditorModel>());
+	readonly onDidChangeContent = this._onDidChangeContent.event;
```

Comment:
@bpasero I added this event on this service so I could detect only when untitled files have changed. Please let me know if this was the wrong change, and I can look for a different way to do this.